config-defaults.sh

```diff
@@ -11,3 +11,4 @@ export SERVICE_NETWORK_CIDR="${SERVICE_NETWORK_CIDR:-${SERVICE_NETWORK}/16}"
 export SERVICE_NETWORK_MASK="${SERVICE_NETWORK_MASK:-255.255.0.0}"
 
 export REPLICAS="${REPLICAS:-1}"
+export NETWORK_TYPE=OpenShiftSDN
```

Comment:
need to follow the previous examples in the file.  only want to override with default if not set in the `config.sh`.
```
export NETWORK_TYPE="${NETWORK_TYPE:-OpenShiftSDN}"
```